packages/dashboard/src/features/auth/components/ResendSettingsCard.tsx

```
@@ -0,0 +1,221 @@
+import { z } from 'zod';
+import { type ResendConfigSchema, type UpsertResendConfigRequest } from '@insforge/shared-schemas';
+
+const resendFormSchema = z
+  .object({
+    enabled: z.boolean(),
+    apiKey: z.string().optional(),
+    senderEmail: z.string(),
+    senderName: z.string(),
+  })
+  .superRefine((data, ctx) => {
+    if (!data.enabled) return;
```

ESLint

[error] Expected { after 'if' condition.

(curly)